db/migrate/20160825122452_remove_duplicate_drafts.rb

```diff
@@ -0,0 +1,34 @@
+class RemoveDuplicateDrafts < ActiveRecord::Migration
+  def up
+    content_items = ContentItem.where(id: [815434, 818621])
+
+    if content_items.any? { |x| x.updated_at != x.created_at }
+      raise "content item #{x.id} has been changed"
```

Comment:
So in the scenario that the data has changed and your assumptions are incorrect
- Raising an error here would fail a production deploy and block us until the migration was altered for the new data
- Just logging that problem occurred (could use errbit or logger.warn rather than just puts) wouldn't block anything

We still need to monitor for this error so even if it failed completely silently we'd still catch it as we need to monitor for similar.